src/components/MayHaveMissed.vue

```diff
@@ -43,7 +45,7 @@
       </a>
     </header>
     <section class="article-body">
-      <pre>{{ articleArray.article_body }}</pre>
+      <pre>{{ articleArray.article_body.trim() }}</pre>
```

Comment:
Smart. Does this take a parameter inline, or done somewhere more globally elsewhere?

Is there a default?

Reply:
I'm not sure if this comment was supposed to be on this line or not but because it is I'll say: eventually i want the trim to happen within choc sauce, but choc sauce is so complex right now it was just easier to add the trim here.